src/core/error_helpers.js

```diff
@@ -253,6 +406,8 @@ function helpForMisusedAtTopLevelCode(e, log) {
 
 // Exposing this primarily for unit testing.
 p5.prototype._helpForMisusedAtTopLevelCode = helpForMisusedAtTopLevelCode;
+p5.prototype._validateParameters = validateParameters;
+p5._validateParameters = validateParameters;
```

Comment:
do we need both lines here?
and can we just put this in the definition above? ie
p5._validateParameters = function(......